src/components/Form/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { FormCombinedProps, isNewProps } from "./types";
+import { default as FormDeprecated } from './Form_deprecated'
+import { Form as FormInternal } from './Form';
+import { FieldValues } from 'react-hook-form';
+
+const Form = <TFieldValues extends FieldValues = FieldValues>(props: FormCombinedProps<TFieldValues>) => {
+  if (isNewProps(props)) {
+    return (<FormInternal {...props} />)
+  }
+
+  return (
```

Comment:
why has to be wrapped in a fragment ?
```
return <FormDeprecated          {...props} />
```
?

Reply:
Originally it was for `// @ts-ignore` but I think that should be possible with a one liner as well